.ci.yaml

```diff
@@ -30,6 +30,7 @@ platform_properties:
       caches: >-
         [
           {"name":"builder_linux_engine","path":"builder"},
+          {"name":"gradle","path":"gradle"},
```

Comment:
would it make sense to document where these are defined in https://github.com/flutter/cocoon/blob/master/CI_YAML.md ?

Reply:
I need to think about this more. I'm looking into a larger overhaul to caches so we can version them, so once I understand them I can add documentation.

\cc @christopherfujino who is working on the single recipe builds